calculator.js

```diff
@@ -1,5 +1,69 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+const exampleInput = {
+  num1: 8,
+  num2: 0.9,
+  operation: 'MULtiplY',
+};
+
+const operations = [
+  'add', '+', 
+  'subtract', '-', 
+  'multiply', '*', 
+  'divide', '/'
+];
+
+
+const isOperationIncluded = (operation) => {
+  return operations.includes(operation);
+};
+
+const areNumbersValid = (num1, num2) => {
+  if (typeof num1 === 'string' || typeof num2 === 'string') {
+    return false;
+  } else if (num1 === null || num2 === null) {
+    return false;
+  } else if (num1 === undefined || num2 === undefined) {
+    return false;
+  } else {
+    return true;
+  }
+};
```

Comment:
I'd recommend `throw`ing an exception in functions that validate things like this.  That way you won't forget to check the return value.